modules/openthread/Kconfig.thread

```
@@ -204,3 +204,9 @@ config OPENTHREAD_NAT64_CIDR
 	  to set source address of the outgoing translated IPv4 packets.
 	  The CIDR must have four bytes in the address with the
 	  non-zero length of prefix (e.g., "127.0.0.1/24").
+
+config OPENTHREAD_STORE_FRAME_COUNTER_AHEAD
+	int "Openthread value ahead of the current frame counter for persistent storage."
+	default 100000
+	help
+	   Openthread value ahead of the current frame counter for persistent storage.
```

Nit, although having the same value for the prompt and description doesn't seem helpful:

```suggestion
	  Openthread value ahead of the current frame counter for persistent storage.
```

---

Thanks, fixed in other places as well.
I've shortened the prompt, leaving the original description the same as OpenThread's description